### PR TITLE
WIP fixes to reviewer concerns

### PR DESCRIPTION
**Pull request by RacheltheEditor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -315,30 +316,32 @@ compensation.
...
   under the laws of a specific state.
-- 🄳 A [**startup**](https://en.wikipedia.org/wiki/Startup_company) is an emerging company that
-  aspires to grow quickly.
+- 🄳 A [**startup**](https://en.wikipedia.org/wiki/Startup_company) is an emerging company,
+  typically a [private company](#public-and-private-companies), that aspires to grow quickly in
+  size, revenue, and influence.
   Once a company is established in the market and successful for a while, it usually stops
   being called a startup.
-  - ☝️ The term *startup* is pretty informal.
+  - ☝️ Vague, huh?
```
Comment (jlevy) [inline README.md]: To my taste, this crosses into the "too informal for this topic" range. It's not really vague, it's just informal—not a legal concept. How about updating to:

"Unlike some of the legal terms we discuss, the term *startup* is informal, and not everyone uses it consistently. As famous investor Paul Graham [emphasizes](http://www.paulgraham.com/growth.html), it's likely best to think of a startup as any [early stage](#stages-of-a-startup) company intending to grow quickly."

Comment (RacheltheEditor) [reply README.md]: I could not find my comments for a revised startup definition anywhere in commits, where I suggested that we used the Paul Graham quote... Any guesses where it ended up? (Also, this is WIP so hang tight before review. Ignore this PR for now.)

Comment (RacheltheEditor) [reply README.md]: corrected!

File: README.md [hunk 2]
```diff
@@ -297,55 +298,68 @@ and it is far easier to work with them when you have deeper understanding yourse
...
+  size, revenue, and influence.
   Once a company is established in the market and successful for a while, it usually stops
   being called a startup.
-  - ☝️ The term *startup* is pretty informal.
-    It’s not a legal concept, and not everyone uses it consistently.
-    For our purposes, it’s best just to think of it as the young period of a fast-growing
-    company.
+  - ☝️ Unlike some of the legal terms we discuss, the term *startup* is informal, and not
+    everyone uses it consistently.
+    As distinguished investor [Paul Graham]\([https://en.wikipedia.org/wiki/Paul*Graham*(programmer](<https://en.wikipedia.org/wiki/Paul_Graham_(programmer>))
```
Comment (jlevy) [inline README.md]: Whole bunch of link formatting issues, likely due to Flowmark on incorrect link syntax. See another wikipedia link in the doc for an exact example.

File: README.md [hunk 3]
```diff
@@ -297,55 +298,68 @@ and it is far easier to work with them when you have deeper understanding yourse
...
+- What is stock, and how does one own it?
+- How does stock relate to shares and equity?
+- What can you do with stock or stock options?
+- What kinds of companies utilize equity compensation?
 
-This requires discussing starting with the basics of how companies and stock work in the
-United States.
-If you think you know all this, you can [jump ahead](#equity-compensation-basics) to
-compensation.
+That last question is often overlooked or misunderstood, and it’s really where everything
```
Comment (jlevy) [inline README.md]: I still am finding this intro confusing. We don't address stock options here so don't want to mention that yet. Also I'm not sure that last question is misunderstood as much as just fundamental. How replacing the first two paragraphs with:

To understand how equity is used for compensation, it's important to understand some fundamentals about how companies are organized and owned. What are stock and shares? How is stock in a company owned? And what kinds of companies use stock for compensation?

File: README.md [hunk 4]
```diff
@@ -297,55 +298,68 @@ and it is far easier to work with them when you have deeper understanding yourse
...
   - 🚧 Groundwork:
     What are good stats on how many people work in startups vs established companies?
 - Companies in the US include sole proprietorships, partnerships, limited liability
   companies (LLCs), S corporations, and C corporations.
   - 🔸 This Guide does not cover equity compensation in LLCs, as they are rarely used as the
-    choice of entity for technology startups, and equity compensation in LLCs is dramatically
-    different from equity compensation in corporations.
-    So we’ll focus on the one type of company most readers of this Guide will be concerned
-    with: The C corporation.
+    choice of entity for startups in technology and in non-tech sectors, and equity
```
Comment (jlevy) [inline README.md]: Still have a few issues with this (it's longer than it needs to be, and it also again makes it sound like this is a startup-only Guide).

Let's just tighten this up to keep it to the point:

This Guide [does not cover](#scope) equity compensation in LLCs, S corporations, partnerships, or sole proprietorships. Equity compensation in each of these kinds of business entities is significantly different than in C corporations.

File: README.md [hunk 5]
```diff
@@ -297,55 +298,68 @@ and it is far easier to work with them when you have deeper understanding yourse
...
+    compensation in LLCs is dramatically different from equity compensation in corporations.
+    The equity components of sole proprietorships, partnerships, and S corps are equally
+    divergent from equity compensation in typical public and private companies, including
+    startups, which are all overwhelmingly C corporations.
 - 🄳 A [**C corporation (or C corp)**](https://en.wikipedia.org/wiki/C_corporation) is a type of
   business entity in the United States.
-  Specifically, it’s a kind of
-  [**stock corporation**](https://en.wikipedia.org/wiki/Joint-stock_company) with certain federal
+  Specifically, it’s a kind of stock corporation (also called a
```
Comment (jlevy) [inline README.md]: This should somehow be merged with the new definition above, as they're essentially alternate terms. Having "joint-stock company" be boldfaced here like a definition here, but not listed as an alternate above is confusing.

I'd figured we would define stock corporation and C corporation at once, but we could put both right here as separate definitions.

File: README.md [hunk 6]
```diff
@@ -297,55 +298,68 @@ and it is far easier to work with them when you have deeper understanding yourse
...
-This requires discussing starting with the basics of how companies and stock work in the
-United States.
-If you think you know all this, you can [jump ahead](#equity-compensation-basics) to
-compensation.
+That last question is often overlooked or misunderstood, and it’s really where everything
+starts.
+
+🄳 For-profit companies that share ownership (in divestments of stock called "shares") in
+exchange for capital or labor are called
+[**stock corporations**](https://www.thebalancesmb.com/what-is-a-stock-corporation-398479). We’ll
```
Comment (jlevy) [inline README.md]: The definitive link for stock corporation is actually the one we had:
https://en.wikipedia.org/wiki/Joint-stock_company

thebalancesmb.com is Google's first match but I don't think it's good enough to merit being the primary link.

Stock corporation and joint-stock company are just alternate terms. We should define them together, probably below.

Possibly we also need a bit of language explaining that company and corporation are essentially synonymous, if that was confusing.

File: README.md [hunk 7]
```diff
@@ -423,54 +437,79 @@ ownership in a company you work for.
...
-    a company raises money and issues stock, the hope and intention is that what you do have
-    is worth more.
-    The size of your slice gets relatively smaller, but if the company is growing, the size of
-    the cake gets bigger.
+- 🄳 As these companies add (or “issue”) stock, the number of outstanding shares goes up,
+  and the percentage ownership of each shareholder goes down.
+  This is called [**dilution**](http://www.investopedia.com/terms/d/dilution.asp).
+  - Dilution doesn’t necessarily mean that you’re losing anything as a shareholder.
+    As a company issues stock and raises money, the smaller percentage of the company you *do*
+    have should be *worth* more.
```
Comment (jlevy) [inline README.md]: should -> could

File: README.md [hunk 8]
```diff
@@ -423,54 +437,79 @@ ownership in a company you work for.
...
-- 🄳 As these companies add (or “issue”) stock, the outstanding shares goes up, and the
-  percentage ownership of each shareholder goes down.
-  This is called [**dilution**](http://www.investopedia.com/terms/d/dilution.asp). That’s a
-  confusing word to most humans, but the concept is relatively simple:
-  - Dilution doesn’t necessarily mean that as a shareholder you’re losing anything-usually, as
-    a company raises money and issues stock, the hope and intention is that what you do have
-    is worth more.
-    The size of your slice gets relatively smaller, but if the company is growing, the size of
-    the cake gets bigger.
+- 🄳 As these companies add (or “issue”) stock, the number of outstanding shares goes up,
```
Comment (jlevy) [inline README.md]: 👍

File: README.md [hunk 9]
```diff
@@ -423,54 +437,79 @@ ownership in a company you work for.
...
     case, since 120%×120%×120% is 173%-and each shareholder owns proportionally less of the
     company.
-- 🄳 The [**valuation**](https://en.wikipedia.org/wiki/Valuation_(finance)) of the company is the
-  present value investors believe the company has.
-  If the company is doing well, making more revenue or gaining users (indicating future
-  revenue), the company’s valuation will usually be going up.
-  That is, the **price** to an investor for buying one share would be increasing.
+- 🄳 The [**valuation**](https://en.wikipedia.org/wiki/Valuation_(finance))
+  of the company is the present value investors believe the company has.
+  If the company is doing well, growing revenue by gaining customers or measuring high
```
Comment (jlevy) [inline README.md]: "measuring high anticipation" sounds strange and unclear to my ear. Let's go with something like:

"growing revenue, gaining customers, or in other ways building evidence that there is a large market for their product in the future"

**Review by jlevy — CHANGES_REQUESTED** [review]
This PR has some great specific improvements but they're mixed in with big additions I don't think are ready to merge. We can DM to sort it out.

File: README.md [hunk 10]
```diff
@@ -437,54 +437,79 @@ ownership in a company you work for.
...
-  shares goes down.
-  When investors expect a company to do better than it actually does, it can have a lot of
-  disappointing consequences for shareholders.
-  The implications of this are complicated, and we’ll get into them further.
-  But it can happen that a company fails and everyone’s shares become worthless, or that
-  certain kinds of shares become worthless while other kinds have some value.
+  shares goes down, because other investors and the public are being told that company is
+  worth less than initially anticipated, and so will be less likely to put their own money
+  into the company or its product.
+- When investors expect a company to do better than it actually does, it’s called
```
Comment (jlevy) [inline README.md]: This is a big addition. Let's talk offline about it.

**Comment by jlevy** [comment]
Changes moved into #116. Closing.

**Pull request closed without merging**